.github/ISSUE_TEMPLATE/BUG-REPORT.yaml

```diff
@@ -0,0 +1,39 @@
+name: Bug Report
+description: File a Corso bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Corso Version?
+      description: What version of Corso are you running?
+      value: "Corso vX.X.X"
```

Comment:
Not sure where this would go, but it is important to know whether or not a successful operation was ever completed within the environment.

Reply:
Hopefully but bug report and logs will make this clear. We need to keep the number of required questions at a minimum and do the work in our product to get as much information as possible. The more questions (friction) we add, the less likely it will be for someone to file a bug report.